ext/bin/load-test/load-test.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+from locust import HttpUser, task, between
+import yaml, json, os
+
+class PuppetDbLoadTest(HttpUser):
+    def response_printer(self, p, response):
+        if response.status_code != 200:
+            print(opts['name'] + "[" + str(response.status_code) + "] failed with:")
+            print(response.text)
```

Comment:
`response.text `doesn't have any relevant data, but `response.error` does. Putting the `status_code` between brackets doesn't really say what it is.
Something in this direction might be a little better:
`print(opts['name'] + " finished with status code: " + str(response.status_code) + ".\nFailed with:")
            print(str(response.error) + "\n")`

Reply:
Updated to a more detailed message.